packages/typography/src/Body/Body.tsx

```diff
@@ -22,12 +23,12 @@ const fontWeights: Record<
   Record<BodyFontWeight, number>
 > = {
   default: {
-    regular: 400,
-    medium: 500,
+    regular: fontWeightTokens.regular,
+    medium: fontWeightTokens.medium,
   },
   strong: {
-    regular: 700,
-    medium: 700,
```

Comment:
nit: there's an opportunity here to type the `weight` prop as `keyof fontWeightTokens` and simplify the CSS on line 49 to:

```
font-weight: ${fontWeightTokens[weight]`
strong, b {
   font-weight: ${fontWeightTokens['bold']
}
```

Reply:
can definitely be stricter about the ts but i believe it'd look like: `(typeof fontWeightTokens)[keyof typeof fontWeightTokens]`

Reply:
actually cleaned it up pretty significantly - can you take another look @spark33?